packages/react/src/Form/MultiStep/FormStepHeader.tsx

```diff
@@ -0,0 +1,110 @@
+import { CheckCircle } from '@obosbbl/grunnmuren-icons';
+import { ReactNode, useMemo } from 'react';
+import { FormStatus } from './FormStep';
+import classNames from 'clsx';
+
+export interface FormStepHeaderProps {
+  className?: string;
+  collapsed: boolean;
+  step: number;
+  onClick: (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
+  children: ReactNode;
+  formStatus: FormStatus;
+}
+
+export const FormStepHeader = (props: FormStepHeaderProps) => {
+  const { className, collapsed, step, onClick, children, formStatus } = props;
+
+  return (
+    <button
+      type="button"
+      aria-label={collapsed ? `Åpne steg ${step}` : `Steg ${step} er åpent`}
```

Comment:
Blir det ikke litt feil å bruke aria-label her? Skjermlesere får lest opp denne teksten i stedet for innholdet i headingen. Vil det ikke være bedre med aria-expanded for å si noe om åpen/lukket state? Da slipper vi språklag her også

Reply:
Vi trenger kanskje ikke aria-expanded her med bruk av aria-disabled. Da sier skjermleseren at knappen er "dimmed" om den er disabled.

Reply:
droppa aria-expanded og aria-label, bruker bare aria-disabled